sourcegraph/sourcegraph
# Cody App about page

Adds a simple about page, so later, we can add update mechanisms to it.

It also removes the Advanced > Updates page and moves its content to the about box.

## Test plan

1. Click on Settings
2. See the version on the About page
